web/src/app/alerts/components/AlertDetails.tsx

```diff
@@ -369,11 +369,13 @@ export default function AlertDetails(props: AlertDetailsProps): JSX.Element {
         <Card sx={{ width: '100%' }}>
           <CardContent data-cy='alert-summary'>
             <Grid container spacing={1}>
-              <Grid item xs={12}>
-                <Typography variant='body1'>
-                  {ServiceLink(alert.service)}
-                </Typography>
-              </Grid>
+              {alert.service && (
```

Comment:
From my understanding of JS, the `&&` should be fine here since Objects are never falsy

docs: https://developer.mozilla.org/en-US/docs/Glossary/Falsy

this should avoid the common gotcha with `&&` and conditional rendering. Since undefined and null won't render and `alert.service` is always truthy

https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator